src/backend/web/routes/user.js

```diff
@@ -16,4 +16,15 @@ router.get('/info', protect(), (req, res) => {
   return res.json(req.user);
 });
 
+router.get('/feeds', protect(), async (req, res) => {
+  const { user } = req;
+  try {
+    const feeds = await user.feeds();
+    return res.status(200).send(feeds);
```

Comment:
Is there a specific reason we're using `res.send()` here instead of `res.json()`? Otherwise, looks good!

Reply:
We looked this up in the express docs, and it does type sniffing to send JSON when you give an Array.  So either is fine.

Reply:
Cool, good to know!

Reply:
Same.  I had the exact reaction you did, and had to go look it up.  I've always used `.json()`